openbas-front/src/admin/components/scenarios/scenario/notification_rule/NotificationRuleForm.tsx

```diff
@@ -96,6 +96,7 @@ const NotificationRuleForm: FunctionComponent<Props> = ({
               onClick={edition ? handleOpenDelete : handleClose}
               style={{ marginRight: theme.spacing(1) }}
               disabled={isSubmitting}
+              color={edition ? 'error' : 'primary'}
```

Comment:
What is our standard? Is it documented somewhere in Notion?
Would it be wise to have a small component for this part of the UI? @guillaumejparis & @MarineLeM what do think ?